### ISSUE TITLE
[Question] Do internal ping messages count for Azure plan limit

### ISSUE THREAD

**Issue by alexiordan** [Azure/azure-signalr#218]
Currently, the Free Plan has a limit of 20K messages/day/unit.

Do internal ping messages count too?

For one client, if enables ping to server each 15 seconds:
2 / minute => 120 / h => 2 880 / day one way.

So both ways for a client will be about 6 000 messages only for the infrastructure...

**Comment by zackliu** [comment]
Yes, the ping message is counted now. We didn't realize the amount is so huge and we're now investigating a better solution.

**Comment by alexiordan** [comment]
Thank you for the acknowledge. Would you please update this issue when you have new information?

I believe this is an issue for the Standard Plans too, so I might suggest to have that in mind too.

**Comment by lbargery** [comment]
Hi, I've just deployed azure signalr (and did it late at night to make sure didn't affect too many users if it went wrong at all). Only 300 clients are connected at this time but I've already used up almost 70k messages. Is this the internal pings? I don't think anyone will actually be interacting with the hub at this time. Just a bit worried about hitting my limits! Have attached photo of the azure metrics. Thanks in advance.

![image](https://user-images.githubusercontent.com/5844039/46922882-715bf300-d007-11e8-84cc-99b382f6c8fe.png)

**Comment by zackliu** [comment]
A good news is we start to stop counting ping message (server -> client or client -> server). But it's won't apply to your existing service immediately automatically. A quick way to approach that is to deploy a new signalr service or you can wait for us updating all the existing services.
@lbargery according to you client count, I thinks it's ping message. You can try the practice above. (Now available in WestUS, EastUS, WestUS2, SoutheastAsia, WestEurope and waiting for a few days for other regions)

**Comment by lbargery** [comment]
@zackliu Thanks this seems to be fine now!
